src/Reanimate/Animation.hs

```diff
@@ -1,4 +1,41 @@
-module Reanimate.Animation where
+module Reanimate.Animation
+  ( Duration
+  , Time
+  , SVG
+  , Animation(..) -- TODO should this be exposed? The constructor is used directly in Effect.hs
```

Comment:
TODO

Reply:
Ideally, it would not be exposed and all other modules would just use the appropriate constructors.

The animation data-type will eventually be extended to support generating slides and PDFs.